Libraries/LibWeb/XHR/FormData.h

```diff
@@ -24,7 +24,7 @@ class FormData : public Bindings::PlatformObject {
 public:
     virtual ~FormData() override;
 
-    static WebIDL::ExceptionOr<GC::Ref<FormData>> construct_impl(JS::Realm&, GC::Ptr<HTML::HTMLFormElement> form = {});
+    static WebIDL::ExceptionOr<GC::Ref<FormData>> construct_impl(JS::Realm&, GC::Ptr<HTML::HTMLFormElement> form = {}, Optional<GC::Ptr<HTML::HTMLElement>> submitter = {});
```

Comment:
You should be able to simplify the implementation slightly by changing the type of submitter to `GC::Ptr<HTML::HTMLElement>` and making its default value `nullptr`.